sklearn/utils/tests/test_fast_dict.py

```diff
@@ -29,3 +30,16 @@ def test_int_float_dict_argmin():
     values = np.arange(100, dtype=np.float64)
     d = IntFloatDict(keys, values)
     assert argmin(d) == (0, 0)
+
+
+def test_to_arrays():
```

Comment:
Perhaps it is worth adding a quick description of what the test is intended to test? E.g. test Cython function for converting integer and float dictionary into a numpy array.

Reply:
Added.